ocaml/idl/datamodel.ml

```diff
@@ -9048,7 +9048,7 @@ let pgpu =
       field ~qualifier:DynamicRO ~ty:(Map (Ref _vgpu_type, Int)) ~lifecycle:[Published, rel_vgpu_productisation, ""] ~default_value:(Some (VMap [])) "supported_VGPU_max_capacities" "A map relating each VGPU type supported on this GPU to the maximum number of VGPUs of that type which can run simultaneously on this GPU";
       field ~qualifier:DynamicRO ~ty:(pgpu_dom0_access) ~lifecycle:[Published, rel_cream, ""] ~default_value:(Some (VEnum "enabled")) "dom0_access" "The accessibility of this device from dom0";
       field ~qualifier:DynamicRO ~ty:Bool ~lifecycle:[Published, rel_cream, ""] ~default_value:(Some (VBool false)) "is_system_display_device" "Is this device the system display device";
-      field ~qualifier:DynamicRO ~ty:(Map (String,String)) ~lifecycle:[Prototyped, rel_vgpu_migration_tech_preview, ""] ~default_value:(Some (VMap [])) "compatibility_metadata" "PGPU metadata to determine whether a VGPU can migrate between two PGPUs";
+      field ~qualifier:DynamicRO ~ty:(Map (String,String)) ~lifecycle:[Prototyped, rel_vgpu_migration_tech_preview, ""] ~default_value:(Some (VMap [])) "compatibility_metadata" "Base64 encoded PGPU metadata to determine whether a VGPU can migrate between two PGPUs";
```

Comment:
I wouldn't say that here. Each entry in the string -> string map can be different.